addons/im_livechat/controllers/main.py

```diff
@@ -157,7 +158,7 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                 "channel_type": "livechat",
                 "chatbot": (
                     {
-                        "script": chatbot_script._format_for_frontend(),
+                        "script": chatbot_script.id,
```

Comment:
We probably want to rename `script_id` then (in all places where we make this kind of change).

Reply:
IMO, we should not rename it to `script_id`. The reason is that on the client side, we would access it as `ob.script_id`, which is less preferred in JavaScript as it implies that it’s an identifier (ID) rather than an object representing a script. Additionally, it’s not a Python model field but a key specifically created for the client-side logic. Keeping it as `script` makes the intent clearer and aligns better with how we typically handle objects in JavaScript.